lib/prediction_class.py

```diff
@@ -52,9 +52,9 @@ def check_length_valid_for_allele(self, length, allele):
 
     def predict(self, input_file, allele, epitope_length, iedb_executable_path, iedb_retries):
         if iedb_executable_path is not None:
-            arguments = self.iedb_executable_params(iedb_executable_path, self.iedb_prediction_method, allele, input_file, epitope_length)
+            arguments = ['python'].extend(self.iedb_executable_params(iedb_executable_path, self.iedb_prediction_method, allele, input_file, epitope_length))
```

Comment:
Would there be a benefit to using [`sys.executable`](https://docs.python.org/3/library/sys.html#sys.executable) here instead of trusting that `python` will be the correct interpreter?

Reply:
That's a good idea I think.